analysis/dataflow/callgraph_122_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build go1.22
+//go:build !go1.22
```

Comment:
Why is this build tag disabled? It might be better to delete the file if its contents are duplicated in callgraph_123_test.go

Reply:
I meant not 1.23, will fix.